scripts/common/preflights.sh

```diff
@@ -657,3 +658,21 @@ function bail_if_unsupported_openebs_to_rook_version() {
     fi
 }
 
+# not allow run the installer/upgrade when kurl version is lower than the previous applied before
+function bail_if_kurl_version_is_lower_previous_config() {
+    if commandExists kubectl; then
+       local current_kurl_version
```

Comment:
in the interest of not losing my previous comment:

in my opinion the variable current_kurl_version is confusing as named and should be named previous_kurl_version

Reply:
It is addressed with your suggestion